2.4/root/usr/bin/run-mongod-replication

```diff
@@ -68,6 +68,7 @@ ${CONTAINER_SCRIPTS_PATH}/replset_supervisor.sh "${1:-}" "$$" &
 # Run `unset_env_vars` and `mongod` in a subshell because
 # MONGODB_ADMIN_PASSWORD should still be defined when the trapped call to
 # `cleanup` references it.
+set -m
```

Comment:
`set -o monitor`? And why do we need it at all? Maybe put a comment here about this too? Looks like a wizardry..